app/src/main/java/com/google/droidjump/models/Droid.java

```diff
@@ -68,9 +69,7 @@ public void setCrouching(boolean crouching) {
         isCrouching = crouching;
     }
 
-    public Bitmap[] getDroidTypes() {
-        return droidTypes;
-    }
+    public Bitmap[] getDroidTypes() { return droidTypes; }
```

Comment:
nit: revert

Reply:
deleted this method, so there is no need to do it